<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ffprocessobj</class>
 <widget class="QDialog" name="ffprocessobj">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>641</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="killclose_btn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>400</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Kill</string>
   </property>
  </widget>
  <widget class="QLabel" name="status_lb">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>411</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Running</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="command_tb">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>400</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>370</y>
     <width>251</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="p_lb">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>380</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0%</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
